tests/testthat/error/test-subscript-loc.txt

```diff
@@ -354,6 +368,11 @@ Error: Can't subset elements beyond the end with non-consecutive locations.
 i The input has size 2.
 x The subscript `foo` contains non-consecutive location 4.
 
+> num_as_location(0, 1, zero = "error", arg = "foo")
+Error: Must subset elements with a valid subscript vector.
+x The subscript `foo` can't contain `0` values.
+i It had a zero value at location 1.
```

Comment:
The other messages use the present "has", this one the past "had". Use "has" everywhere?